### ISSUE TITLE
CNV.Discovery.sh - use of uninitialized value message

### ISSUE THREAD

**Issue by M-nied7** [JiangYuLab/CNVcaller#10]
Hello,

Could you please help me with the following issue? What can be a reason for this error when running CNV.Discovery.sh?

> Use of uninitialized value $sd in numeric gt (>) at /CNVcaller/bin/2.1.CNVDiscoveryMerge.pl line 72, <IN> line 10.
> 2.1179679144385

Where should I look for a root cause?

Thank you for sharing your software!

**Comment by biozzq** [comment]
Hi @M-nied7 

Could you share me the normalized read depth file list? It seems that some of your normalized filenames are out of line.

Best wishes,
Zheng Zhuqing

**Comment by M-nied7** [comment]
Hi, thank you for your reply! I attached the list. I am still testing demo dataset before moving to my data. 
[List_normalized_RD.txt](https://github.com/JiangYuLab/CNVcaller/files/5519780/List_normalized_RD.txt)

And ls from the specified folder is:
ERR340328_mean_56.58_SD_9.35_sex_1
ERR340329_mean_49.92_SD_9.10_sex_1
ERR340330_mean_108.90_SD_12.24_sex_1
ERR340331_mean_102.90_SD_13.05_sex_1
ERR340333_mean_54.45_SD_8.73_sex_1
ERR340335_mean_58.20_SD_8.76_sex_1
ERR340336_mean_57.23_SD_8.65_sex_1
ERR340338_mean_97.00_SD_11.30_sex_1
ERR340340_mean_106.08_SD_13.01_sex_1

I am a bit lost...

**Comment by M-nied7** [comment]
I realized I forgot to add that in comparison to demo, I have changed the window size to 600, but I did it consistently through all steps.

**Comment by M-nied7** [comment]
Hi @biozzq I have realized my mistake! I changed the window size to 600 but I used the default dupfile provided for the demo data. I haven't realized that the duplicated window record file also depends on the window size (which makes prefect sense now!). I am a bit surprised I hadn't seen an error on earlier step. Sorry for bothering you! I hope this issue helps someone else to understand their problem.
